dll/cpl/sysdm/virtmem.c

```diff
@@ -569,27 +577,45 @@ OnSelChange(HWND hwndDlg, PVIRTMEM pVirtMem)
         }
 
         /* Set minimum pagefile size */
-        SetDlgItemText(hwndDlg, IDC_MINIMUM, _T("16 MB"));
+        SetDlgItemText(hwndDlg, IDC_MINIMUM, _T("2 MB"));
 
         /* Set recommended pagefile size */
         MemoryStatus.dwLength = sizeof(MEMORYSTATUSEX);
         if (GlobalMemoryStatusEx(&MemoryStatus))
         {
             FreeMemMb = (UINT)(MemoryStatus.ullTotalPhys / (1024 * 1024));
-            _stprintf(szBuffer, _T("%u MB"), FreeMemMb + (FreeMemMb / 2));
+            RecoMemMb = FreeMemMb + (FreeMemMb / 2);
+            if (RecoMemMb > 4096)
+                RecoMemMb = 4096;
```

Comment:
Where does this come from ? Do you have tests for this ? Also, the suggestion of @katahiromz is also reused here :D

Reply:
This was present before the change. Recommended VM size was 150% x Free memory.
I kept this unchanged. Looks similar to Windows settings